examples/notebooks/beam-ml/run_inference_pytorch.ipynb

```diff
@@ -84,6 +84,13 @@
       "metadata": {
         "id": "loxD-rOVchRn"
       },
+      "outputs": [],
+      "source": [
+        "!pip install apache_beam[gcp,dataframe] --quiet"
+      ],
+      "metadata": {
+        "id": "loxD-rOVchRn"
+      },
```

Comment:
Was this change intentional? It looks like it no-ops

Reply:
I think this is a change that Anand made, and for some reason it's showing up in this PR: https://github.com/apache/beam/pull/24336/files#diff-d74554e3521bce6230681b255694c89efcceae49a13c2baccb9b91d983b1d6fc

I can try to get rid of it again, but it keeps showing up.

Reply:
The change itself is good, but source/metadata/outputs are duplicated in your version right now

Reply:
Probably just a weird merge in of master, you should just be able to delete these lines though

Reply:
I think this was a bad merge after a conflict. I'll revert it.

Reply:
I think I successfully got rid of this.